common/script/index.coffee

```diff
@@ -1692,8 +1809,16 @@ api.wrap = (user, main=true) ->
       # Analytics
       user.flags.cronCount?=0
       user.flags.cronCount++
-      options.mixpanel?.track('Cron',{'distinct_id':user._id,'resting':user.preferences.sleep})
-      options.ga?.event('behavior', 'cron', 'cron', user.flags.cronCount).send(); #TODO userId for cohort
+
+      analyticsData = {
+        category: 'behavior',
+        gaLabel: user.flags.cronCount,
```

Comment:
I think I interpreted the value param as value in dollars, but I guess that's not what it is for. (That's how we use it for purchases, I believe)

I don't think my track code has a way to track value. Thinking of switching to the params syntax instead of arguments: https://github.com/peaksandpies/universal-analytics#event-tracking